pkg/k2/testdata/bucket_ro/my-bucket.resources.yaml

```diff
@@ -8,9 +8,9 @@ resources:
             RESOURCE_NAME: my-bucket
 edges:
 outputs:
+    Bucket:
```

Comment:
These changes to outputs are suspicious - is there some nondeterminism here?

Reply:
very possible, but didn't investigate too deeply